\documentclass[11pt,a4paper,sans]{moderncv}

% moderncv themes
\moderncvstyle{classic} % casual, classic, banking, oldstyle and fancy
\moderncvcolor{blue} 
\moderncvhead{3}

\usepackage[utf8]{inputenc}
\usepackage[scale=0.75, top=2cm]{geometry}
%\setlength{\hintscolumnwidth}{3cm} 
%\settowidth{\hintscolumnwidth}{Jun-Sep, 2011} 
%\setlength{\makecvtitlenamewidth}{10cm} % for the 'classic' style

% personal data
\name{Diego}{Taquiri-Diaz}
%\title{Resumé title}
%\address{street and number}{postcode city}{country}
\social[github]{diego-taquiri}
\email{diego.taquiri@upch.pe}
\social[twitter]{diego\_taquiri}
%\social[linkedin]{diegotaquiri}
%\phone[mobile]{+51~949~503~024}
%\phone[fixed]{+2~(345)~678~901}
%\phone[fax]{+3~(456)~789~012}

%\homepage{www.johndoe.com}


%\extrainfo{additional information}
%\photo[64pt][0.4pt]{example-image-a}
%\quote{Some quote}

\setlength{\footskip}{66pt}


\begin{document}

\makecvtitle
\vspace{-1cm}

\section{Education}
\cventry{2020--2024}{B.Sc in Biology}
{\footnotesize Universidad Peruana Cayetano Heredia (\#1 program in Peru)}{}{}
{\footnotesize \textit{GPA: 4.0/4.0, 16.3/20.0 in original scale (Top 20\% of the class)}.}  % arguments 3 to 6 can be left empty
\cventry{2017--2019}{B.S.E in Biomedical Engineering}
  %Universidad Peruana Cayetano Heredia (UPCH) y
  {\footnotesize Pontificia Universidad Católica del Perú}{Peru}{}
  {\footnotesize \textit{GPA: 3.9/4.0, 15.5/20.0 in original scale (Top 10\% of the class)}. Transitioned to a B.S. in Biology after 3 years of coursework.}
%, 1st place in class ranking. Completed 3 out of 5 years of coursework. Transitioned to a B.S. in Biology.

\section{Skills}

\cvitem{}{\textbf{Programming:} 
\footnotesize Python, R/RStudio, Nextflow, HPC, AWS Cloud, Git/GitHub, Bash, Unix. 
}
\cvitem{}{\textbf{Machine Learning:} 
\footnotesize PyTorch, Transformers, Large Language Models (LLMs), gLMs (genomic), pLMs (protein), CNN, XGBoost, NumPy, Pandas, Polars, Matplotlib, Seaborn.
}
\cvitem{}{\textbf{Toolbox:} 
\footnotesize Visual Studio Code, Cursor AI, Jupyter, SSH, Tidyverse, Docker, LaTeX, ChatGPT.
}
\cvitem{}{\textbf{Bioinformatics:} 
\footnotesize scRNAseq, RNAseq, Genomics, Metagenomics, Phylogenetics, Methylomics, NGS, Biostatistics, Molecular docking, 3D Structure Prediction. 
}
\cvitem{}{\textbf{Languages:} 
\footnotesize English (Full professional proficiency), Spanish (native speaker).
}

\section{Research Experience}
\cventry{2023--Present}{Research Assistant}
  {Grandjean Research Group}{University College London, UK}{}
  {Principal investigator: MD, Ph.D. Louis Grandjean
  \begin{itemize}%
    \item \textbf{Data integration:} {\footnotesize Performed data normalization, transformation, batch effect correction, and dimensionality reduction to characterize the profile of pathogens and antibiotic resistance in the wastewater microbiome from multiple hospitals and temporal points.}
    \item \textbf{Cis-regulatory element prediction:} {\footnotesize Predicted cis-regulatory elements (promoters) of antibiotic resistance genes in bacterial strains and compared mutations across strains to explain differences in phenotypic resistance, despite identical genomic resistance profiles.}
    \item \textbf{Metagenomic assembly:} {\footnotesize Conducted metagenomic assembly of wastewater samples using deep sequencing with PromethION Nanopore technology to generate high-quality MAGs and achieve reproducible detection of bacterial pathogens and antibiotic resistance.}
    \item \textbf{Phylogenetics analysis:} {\footnotesize Annotated gene markers from metagenome-assembled genomes (MAGs), performed multiple sequence alignment, constructed phylogenetic trees, and conducted species identification. Validated candidate novel species using contamination and completeness metrics for quality control.}
    \item \textbf{Methylomic analysis:} {\footnotesize Performed basecalling and alignment using Nanopore sequencing modification models to identify 6mA, 5mC, and 5hmC methylation patterns in \textit{Klebsiella pneumoniae} isolates, analyzing methylation profiles in resistance genes to explore their potential role in antibiotic resistance.}
  \end{itemize}
  }

\cventry{2023--Present}{Research Assistant}
  {Bioinformatics and Molecular Biology Lab}{Universidad Peruana Cayetano Heredia, Peru}
  {}
  {Principal investigator: Ph.D. Mirko Zimic and  Ph.D. Patricia Sheen%
  \begin{itemize}%
    \item \textbf{RNA-seq analysis:} {\footnotesize Performed RNA sequencing analysis of platelets from patients with and without tuberculosis, including splice-aware mapping, unsupervised clustering, differential gene expression, gene ontology, gene set enrichment analysis, and supervised machine learning to predict TB diagnosis based on RNA profiles.}
  \end{itemize}
  }

  \cventry{}{}
  {}{}{}
  {\begin{itemize}%
    \item \textbf{Annotation:} {\footnotesize Conducted functional annotation to identify genes, metabolic pathways, plasmids, and viral elements associated with microbiome changes in metagenomic agricultural soil samples treated with organic fertilizers.}
    \item \textbf{Differential abundance analysis:} {\footnotesize Conducted statistical analysis to identify differentially abundant resistant genes and species associated with caries compared to healthy controls in the oral microbiomes of children, including taxonomic classification and multivariate analysis (PCoA) to effectively cluster microbiome profiles.}
    \item \textbf{Variant calling:} {\footnotesize Conducted reference-based mapping and variant calling on 3,932 \textit{M. tuberculosis} whole genome sequences, followed by heteroresistance analysis to detect low-frequency mutations in bacterial subpopulations across 14 antibiotic resistance genes.}
    \item \textbf{Molecular docking and structural modeling:} {\footnotesize Conducted structural characterization of the PonA1 protein of \textit{M. tuberculosis} using AlphaFold2/ESMFold to predict the full protein structure, followed by deep learning-based (Diffdock) and classical docking (Gnina) methods to identify rifampicin-binding pockets.}
    \item \textbf{Whole-genome analysis:} {\footnotesize Developed an end-to-end Nextflow pipeline to process 187 \textit{M. tuberculosis} genomes sequenced in-house using Nanopore technology, generating post-sequencing metrics such as quality and mapping statistics, and conducted statistical comparisons of extraction methods using R.}
    \item \textbf{TB detection and resistance profiling:} {\footnotesize Developed and evaluated a nanopore sequencing protocol for direct detection of \textit{M. tuberculosis} and antibiotic resistance from sputum samples, utilizing shotgun metagenomics, amplicon sequencing, adaptive sampling, and culture-based whole genome sequencing (WGS). Conducted read filtering, gene depth analysis, resistance profiling across 13 genes, compared enrichment methods, and performed TB lineage profiling.}
    \item \textbf{Proteomic analysis:} {\footnotesize Developed and validated an assay using MALDI-TOF mass spectrometry to detect pyrazinamide resistance in \textit{M. tuberculosis} by analyzing the conversion of pyrazinamide to pyrazinoic acid in MODS culture.}
    \item \textbf{Miscellaneuos:}
    \begin{itemize}
      \item \textbf{Established genomics operations:} {\footnotesize Built the lab's bioinformatics capacity from the ground up after acquiring a nanopore sequencing device, managing everything from experimental design and sequencing monitoring to data processing, analysis, visualization, and interpretation.}
      \item \textbf{Computational Infrastructure Management:} {\footnotesize Managed and optimized the bioinformatics computational infrastructure for a genomics lab, including high-performance computing workstations, large-scale storage systems, and data pipelines.}
      \item \textbf{Team leadership and mentoring:} {\footnotesize Led and mentored a seven-person bioinformatics team, managing multiple projects concurrently.}
      \end{itemize}
  \end{itemize}
  }

%\cventry{2024--Present}{Bachelor’s Thesis}
%  {Bioinformatics and Molecular Biology Lab}{Universidad Peruana Cayetano Heredia, Peru}
%  {}
%  {Principal investigator: Ph.D. Mirko Zimic and  Ph.D. Patricia Sheen\newline{}%
% Adaptive sampling
%  }

\section{GitHub Repositories}
    \cventry{2025}{\href{https://github.com/diego-taquiri/Spatial-Transcriptomics-MERFISH}{Spatial-Transcriptomics-MERFISH}}
    {{\footnotesize Conducted spatial transcriptomics analysis of the MERFISH Mouse Brain Receptor Map dataset, profiling ~100,000 cells and ~500 genes. Used Squidpy and Scanpy to preprocess and analyze single-cell spatial gene expression, applying Leiden clustering, UMAP projections, and marker gene visualization to characterize brain tissue organization and function.}}{}{}  
    {}
    \cventry{2025}{\href{https://github.com/diego-taquiri/scRNAseq_zebrafish_analysis}{scRNAseq Zebrafish Analysis}}
    {{\footnotesize Reproduced single-cell RNA sequencing (scRNA-seq) analysis of endothelial cells in zebrafish embryos under normal and high glucose conditions. Implemented quality control, clustering, batch-effect correction with Harmony, and differential gene expression analysis using Seurat in R. Identified key proangiogenic gene expression changes associated with high glucose exposure and their effects on endothelial cell populations.}}{}{}
    {}
    \cventry{2024}{\href{https://github.com/diego-taquiri/nanoGPT-DNA}{nanoGPT-DNA}}
    {{\footnotesize Developed a lightweight GPT-like large language model (LLM) transformer in PyTorch to learn the regulatory language of DNA sequences. Adapted nanoGPT to train on the human genome with nucleotide-level tokenization, focusing on regulatory motif classification evaluation as a genomic language model (gLM).}}{}{}
    {}
    \cventry{2024}{\href{https://github.com/diego-taquiri/HLCA-scAtlas-Exploration}{HLCA-scAtlas-Exploration}}
    {{\footnotesize Implemented the integration of 13 single-cell RNA sequencing (scRNAseq) datasets from the Human Lung Cell Atlas (HLCA), encompassing ~600,000 cells. Focused on evaluating the effects of normalization, batch-effect correction, and clustering using Scanpy, scVI-tools and Polars.}}{}{}
    {}
    \cventry{2024}{\href{https://github.com/diego-taquiri/EGFR-pLM-Pathogenicity}{EGFR-pLM-Pathogenicity}}
    {{\footnotesize Analyzed EGFR mutations and their impact on therapeutic antibody resistance using structural modeling and protein language models (pLM). Integrated patient-specific EGFR mutations with ESM1b predictions to explore variant pathogenicity and their roles in therapeutic evasion.}}{}{}
    {}
    %Conducted an exploration of single-cell RNA sequencing dataset integration using the Human Lung Cell Atlas (HLCA) as a case study, focusing on the integration of 13 datasets from healthy lung studies and investigating how normalization and batch-effect correction affect clustering, using Polars and Scanpy (Work in progress).
  %\cventry{2024}{\href{https://github.com/diego-taquiri/AWS-nanoGPT-training}{AWS-nanoGPT-training}}
   % {Deployed and trained a small-scale transformer GPT-2 model using PyTorch on AWS infrastructure.}{}{}
    %{}
  \cventry{2024}{\href{https://github.com/diego-taquiri/AWS-scRNAseq-Nextflow}{AWS-scRNAseq-Nextflow}}
    {{\footnotesize Deployed and ran Nextflow-based \texttt{nf-core} pipelines for single-cell RNA sequencing on AWS using AWS Batch and Tower. Automated quality control, integration, clustering, and cell type annotation.}}{}{}
    {}
  \cventry{2024}{\href{https://github.com/diego-taquiri/HeartDiseaseMLInterpretation}{HeartDiseaseMLInterpretation}}
    {{\footnotesize Trained a machine learning model using XGBoost with hyperparameter tuning and implemented SHAP for interpretability, highlighting feature importance in heart disease prediction.}}{}{}
    {}  
  \cventry{2024}{\href{https://github.com/diego-taquiri/AutismSketchClassifier}{AutismSketchClassifier}}
    {{\footnotesize Pre-trained a ResNet CNN neural network and used its feature vectors for KNN classification to detect autism-specific features in children's sketches.}}{}{}
    {}
  \cventry{2024}{\href{https://github.com/diego-taquiri/ECGMortalityPredictor}{ECGMortalityPredictor}}
    {{\footnotesize Processed ECG, EEG, and EMG signals, applying wavelet transforms for denoising and feature extraction. Developed a TinyML-based early warning system using XGBoost to predict cardiovascular risk in Chagas patients.}}{}{}
    {}
  \cventry{2024}{\href{https://github.com/diego-taquiri/SimpleGenomicNextflow}{SimpleGenomicNextflow}}
    {{\footnotesize Developed and maintained a suite of user-friendly and flexible Nextflow scripts for genomic and metagenomic analysis.}}{}{}
    {}
  \cventry{2023}{\href{https://github.com/diego-taquiri/ONT-tb-extraction}{ONT-tb-extraction}}
    {{\footnotesize Developed R scripts for statistical analysis and plotting, along with a Nextflow pipeline for comparative analysis of Nanopore sequencing of TB.}}{}{}
    {}

\section{Publications}
\subsection{Under review}
\cvitem{}{{\footnotesize K. Vallejos-Sánchez, \textbf{\underline{D.A. Taquiri-Díaz}}, O. Romero, A.P. Vargas, J. Coronel, A. Torres, J.L. Perez, A. Ochoa, R.H. Gilman, L. Grandjean, M. Cohen-Gonsaud, M. Zimic, P. Sheen. 
"Identifying heteroresistant tuberculosis infection from whole genome analysis of Peruvian isolates".
%\textit{J. Med. Microbiol.}
\textbf{Contribution: I designed and performed computational analyses.}}}

\subsection{To be submitted in early 2025}
\cvitem{}{{\footnotesize C. León, A. Osmaston, \textbf{\underline{D.A. Taquiri-Díaz}}, O. Romero, J. Perez, B. Sobkowiak, J. Hatcher, A. Torres, R. Gilman, S. Huaman, J. Coronel, M. Zimic, P. Sheen, L. Grandjean.
"A comparison of methods for \textit{Mycobacterium tuberculosis} DNA extraction optimised for long-read Nanopore sequencing." 
%\textit{J. Clin. Microbiol.} 
\textbf{Contribution: I designed and performed computational analyses.}}}

\cvitem{}{{\footnotesize G. Lawson, G. Tan, C. Leon Palomino, A. Osmaston, O. Romero, \textbf{\underline{D.A. Taquiri-Díaz}}, L. Mascaro Rivera, L. Merino Castaneda, I. Baltas, A. Torres Ortiz, B. Sobkowiak, A. Mendoza Ticona, R. Gavilan, L. Alvarado Ruis, D. Gómez de la Torre, R.H. Gilman, J. Hatcher, P. Sheen Cortavarria, M. Zimic Peralta, M. Pajuelo Travezaño, L. Grandjean. 
"Identifying Novel Mechanisms of Carbapenem Resistant Enterobacterales in Lima, Peru". 
\textbf{Contribution: I designed and performed computational analyses, and contributed to manuscript writing.}}}

%\cvitem{}{{\footnotesize B. Balta, A. Zimic-Sheen, \textbf{\underline{D.A. Taquiri-Díaz}}, O. Romero, J. Orellana, A. Chalco, M. Zimic. 
%"Evaluation of the impact of the application of inorganic fertilizers in conjunction with biostimulants on the taxonomic and functional changes of the rhizosphere microbiome of avocado in an agro-exporting farm". 
%\textbf{Contribution: I designed and performed computational analyses, and was involved in writing the paper.}}}

\section{Relevant Coursework}
\cventry{2024}{Natural language processing}
  {{\footnotesize Undergradutate Course (+60 hours)}}
  {{\footnotesize Universidad Peruana Cayetano Heredia, Peru.}}
  {}
  {}
\cventry{2024}{Computer Vision}
  {{\footnotesize Undergradutate Course (+60 hours)}}
  {{\footnotesize Universidad Peruana Cayetano Heredia, Peru.}}
  {}
  {}
\cventry{2024}{Population genomics}
  {{\footnotesize International Workshop (18 hours)}}
  {{\footnotesize Universidad San Martín de Porres, Peru.}}
  {{\footnotesize In collaboration with the Barreiro Lab, University of Chicago.}}
  {}
  {}
\cventry{2023}{Bioinformatics and Artificial Intelligence}
  {{\footnotesize International Training (20 hours)}}
  {{\footnotesize Peruvian Society of Bioinformatics and Computational Biology (SPBBC).}}
  {}
  {}  
\cventry{2023}{Bioinformatics I}
  {{\footnotesize Undergradutate Course (+70 hours)}}
  {{\footnotesize Universidad Peruana Cayetano Heredia, Peru.}}
  {}
  {}
\cventry{2023}{Introduction to Machine Learning}
  {{\footnotesize Undergradutate Course (+50 hours)}}
  {{\footnotesize Universidad Peruana Cayetano Heredia, Peru.}}
  {}
  {}
\cventry{2022}{Scientific Computing}
  {{\footnotesize Undergradutate Course (+60 hours)}}
  {{\footnotesize Universidad Peruana Cayetano Heredia, Peru.}}
  {}
  {}
\cventry{2022}{Neuromatch Academy: Deep Learning}
  {{\footnotesize International Summer School (+60 hours)}}
  {}
  {}
  {}
\cventry{2020}{Advanced Programming}
  {{\footnotesize Undergradutate Course (+60 hours)}}
  {{\footnotesize Universidad Peruana Cayetano Heredia, Peru.}}
  {}
  {}
\cventry{2019}{Fundamentals of Programming}
  {{\footnotesize Undergradutate Course (+60 hours)}}
  {{\footnotesize Universidad Peruana Cayetano Heredia, Peru.}}
  {}
  {}
%\cventry{2023--2024}{Deep learning specialization}
%\cventry{2023--2024}{Computer Vision}

\section{Leadership Experience}
\cventry{2022}{Directive Board Member}
  {Journal Club}{Student Club}{UPCH}
  {{\footnotesize Directed communications and club leader recruitment, successfully establishing 20 specialized journal clubs and guiding over 100 new participants.}}
\cventry{2022}{Research Secretary}
  {Student Center for Sciences CEC}{Student council}
  {UPCH}
  {{\footnotesize Orchestrated the faculty-wide university Science Week, managing over \$3,000 in funding and achieving an engagement of 1,000+ attendees. Additionally, organized a series of science webinars and career guidance sessions, featuring insights from invited speakers.}}
\cventry{2019}{Vice-President}
  {IEEE Student Branch UPCH}{}
  {UPCH}
  {Coordinated multiple university, inter-university, and national congresses, meetings and events, engaging over 100 participants per event.
  }
\cventry{2019}{Founding Leader}
  {Biomedical Engineering Association}{Student council}
  {PUCP}
  {Led the foundational efforts to establish the Biomedical Engineering Association, managed a core group of 10 members in the structuring and drafting of the association’s statutes.
  }



\section{Grant Writing}
\cventry{2023}{Research grant 82878, \$100,000}
  {National Council for Science, Technology, and Technological Innovation (CONCYTEC)}{Peru}
  {}
  {Conceptualized and authored the bioinformatics section of the grant proposal: "Development and evaluation of a MinION (Nanopore) sequencing-based protocol for determining
Heteroresistance in tuberculosis patients directly from sputum samples".
}


\section{Posters \& Presentations}
\cventry{2024}{HeLa Project, Data Science Division, Universidad Peruana Cayetano Heredia}
  {{\footnotesize Invited Seminar}}
  {{\footnotesize Single-cell RNA-seq 101: Fundamentals of Bioinformatics Analysis of Cellular Transcriptomics. \textbf{\underline{D.A. Taquiri-Díaz}}.}}{}
  {}
\cventry{2024}{American Society of Tropical Medicine and Hygiene (ASTMH)}
  {{\footnotesize Accepted Poster: Identifying Novel Mechanisms of Carbapenem Resistant Enterobacterales in Lima, Peru}}
  {{\footnotesize New Orleans, LA, USA (Accepted, not presented due to funding constraints). C. Tan, \textbf{\underline{D.A. Taquiri-Díaz}}, O. Romero.}}{}
  {}
\cventry{2024}{V International Congress of the Peruvian Society of Bioinformatics and Computational Biology:}
  {{\footnotesize Oral Presentation (online): Exploring the Computational Resources of the Bioinformatics and Molecular Biology Laboratory of UPCH and its Applications in Research}}
  {{\footnotesize Sociedad Peruana de Bioinformática y Biología Computacional, Lima, Peru. \textbf{\underline{D.A. Taquiri-Díaz}}.}}{}
  {}
\cventry{2019}{International Conference on Electronics, Electrical Engineering and Computing (XXVI INTERCON)}
  {{\footnotesize Tech Fair Stand: Customized Glove for De Quervain's Tenosynovitis Prevention}}
  {{\footnotesize Universidad Autonoma del Peru, Peru. \textbf{\underline{D.A. Taquiri-Díaz}}, A. Tecse.}}{}
  {}


\section{Honors \& Awards}
\cventry{2019}{International Conference on Electronics, Electrical Engineering and Computing (XXVI INTERCON)}
  {{\footnotesize Best Applied Technological Development}}
  {{\footnotesize Universidad Autonoma del Peru, Peru}}{}
  {}
\cventry{2019}{Institute of Electrical and Electronics Engineers (IEEE)}
  {{\footnotesize Best new student branch}}
  {{\footnotesize Universidad Peruana Cayetano Heredia, Peru.}}{}
  {}
%\cventry{2017}{Universidad Oberta de Cataluña (UOC)}
%{"Internationalization at Home" scholarship (\$500)}{Entrepreneurial Initiative Program.}{}
%{}

\section{Teaching Experience}
  \cventry{2023}{Teaching Assistant}
    {Course: Bioinformatics I: Sequence Analysis}{Master’s Program}{Universidad Peruana Cayetano Heredia, Peru}
    {Led practical workshops (12 hours) for approximately 30 students, covering genomics assembly, molecular modeling with AlphaFold2/3 and molecular docking.
    }
  \cventry{2022}{Academic Tutor}
    {Course: Molecular Biology of the Cell}{Undergraduate’s Program}{Universidad Peruana Cayetano Heredia, Peru}
    {Lectured sessions for the Peer Academic Mentoring Program (37 hours). 
    }

\section{Extracurriculars}

\cventry{2024}{Journal Club Coordinator}
    {Bioinformatics}{Sociedad Peruana de Bioinformática y Biología Computacional}{}
    {}
\cventry{2022}{Journal Club Participant}
    {Structural Biology}{Sociedad Peruana de Bioinformática y Biología Computacional}{}
    {}
\cventry{2022}{Journal Club Coordinator}
    {Artificial Intelligence}{Journal Club UPCH}{}
    {}
\cventry{2021}{Writer}
    {University Journal}{The Novice Scientist UPCH}{}
    {}
\cventry{2020}{Journal Club Participant}
    {Cell Biology}{Journal Club UPCH}{}
    {}
\cventry{2020}{Journal Club Participant}
    {Cancer Biology}{Journal Club UPCH}{}
    {}
  

\section{References}
\cventry{}{MD, Ph.D. Louis Grandjean}
    {Professor of Infectious Diseases at University College London, UK.}{Email: l.grandjean@ucl.ac.uk}{}
    {}
\cventry{}{MD, Ph.D. Robert H. Gilman}
    {Professor of International Health at Johns Hopkins, USA}{Email: rgilman1@jhmi.edu}{}
    {}
\cventry{}{Ph.D. Mirko Zimic}
    {Professor of Bionformatics at Universidad Peruana Cayetano Heredia, Peru}{Email: mirko.zimic@upch.pe}{}
    {}
\cventry{}{Ph.D. Patricia Sheen}
    {Professor of Molecular Biology at Universidad Peruana Cayetano Heredia, Peru}{Email: patricia.sheen@upch.pe}{}
    {}

%\clearpage

\end{document}